node/builder_miner.go

```diff
@@ -53,11 +53,7 @@ func ConfigStorageMiner(c interface{}) Option {
 	return Options(
 
 		Override(new(v1api.FullNode), modules.MakeUuidWrapper),
-		// Needed to instantiate pubsub used by index provider via ConfigCommon
-		Override(new(dtypes.DrandSchedule), modules.BuiltinDrandConfig),
-		Override(new(dtypes.BootstrapPeers), modules.BuiltinBootstrap),
-		Override(new(dtypes.DrandBootstrap), modules.DrandBootstrap),
-		ConfigCommon(&cfg.Common, build.NodeUserVersion(), false),
```

Comment:
@ribasushi Why are we getting rid of these 3 in the Miner ?

Override(new(dtypes.DrandSchedule), modules.BuiltinDrandConfig),
		Override(new(dtypes.BootstrapPeers), modules.BuiltinBootstrap),
		Override(new(dtypes.DrandBootstrap), modules.DrandBootstrap),
		
		Dosen't a Miner need Drand ?